Description: Add faust include dir to CFLAGS to fix FTBS.
Author: Alessandro Ghedini <al3xbio@gmail.com>
Bug-Debian: http://bugs.debian.org/642662
Origin: vendor
Forwarded: http://code.google.com/p/foo-yc20/issues/detail?id=4
Last-Update: 2011-11-01

--- foo-yc20-1.3.0.orig/Makefile
+++ foo-yc20-1.3.0/Makefile
@@ -21,7 +21,7 @@ CFLAGS=-O3 -mtune=native -march=native -
 endif
 endif
 
-CFLAGS_X = $(CFLAGS) -fPIC -DVERSION=$(VERSION) -Isrc/ -Iinclude/ -DPREFIX=$(PREFIX) -Wall
+CFLAGS_X = $(CFLAGS) -fPIC -DVERSION=$(VERSION) -I/usr/lib/faust -Isrc/ -Iinclude/ -DPREFIX=$(PREFIX) -Wall
 
 $(OBJS_NODEPS): CFLAGS_use = $(CFLAGS_X) 
 $(OBJS_JACK): CFLAGS_use = $(CFLAGS_X) `pkg-config --cflags jack`
